python/lancedb/query.py

```diff
@@ -133,11 +169,11 @@ def __init__(self, table: "lancedb.table.Table"):
         deprecated_in="0.3.1",
         removed_in="0.4.0",
         current_version=__version__,
-        details="Use the bar function instead",
+        details="Use to_pandas() instead",
     )
     def to_df(self) -> "pd.DataFrame":
         """
-        Deprecated alias for `to_pandas()`. Please use `to_pandas()` instead.
+        *Deprecated alias for `to_pandas()`. Please use `to_pandas()` instead.*
```

Comment:
Do we need to add a python deprecation warning in the code?

Reply:
Also should we mention when will we remove this function? (i.e., we will move in in version 1.0) or sth. So it wont be here infinitely .

Reply:
we have this decorator:
`@deprecation.deprecated(
        deprecated_in="0.3.1",
        removed_in="0.4.0",
        current_version=__version__,
        details="Use to_pandas() instead",
    )`